.gitignore

```diff
@@ -8,3 +8,6 @@ fuzz/hfuzz_workspace
 #IntelliJ project files
 .idea
 *.iml
+
+#Vscode project files
+.vscode
```

Comment:
Heh, sure, ACK this, though I seem to recall arguments about it every time somebody tries to do this.

Reply:
Don't mind removing this commit although I don't see why this is an issue.